mailbox/api/src/main/java/org/apache/james/mailbox/MessageIdManager.java

```diff
@@ -39,4 +39,6 @@
     void delete(MessageId messageId, List<MailboxId> mailboxIds, MailboxSession mailboxSession) throws MailboxException;
```

Comment:
No, MessageManager should support it, not MessageIdManager

(sorry if description of the ticket contained one error)